packages/webapp/package.json

```diff
@@ -44,7 +40,14 @@
         "react-i18next": "11.18.6",
         "react-icons": "4.7.1",
         "react-redux": "7.2.8",
-        "redux": "4.2.0"
+        "redux": "4.2.0",
```

Comment:
The `devDependencies` have been alphabetically ordered. Can we have that again? Some package managers re-sort them and then it is easier if they are already in order.

Reply:
Done.